lib/flowViewport/sceneIndex/fvpBlockPrimRemovalPropagationSceneIndex.h

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright 2024 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#ifndef FLOW_VIEWPORT_SCENEINDEX_FVP_BLOCK_PRIM_REMOVAL_PROPAGATION_SCENE_INDEX_H
+#define FLOW_VIEWPORT_SCENEINDEX_FVP_BLOCK_PRIM_REMOVAL_PROPAGATION_SCENE_INDEX_H
+
+//Local headers
+#include "flowViewport/api.h"
+#include "flowViewport/sceneIndex/fvpSceneIndexUtils.h"
+#include "flowViewport/selection/fvpSelectionFwd.h"
+#include "flowViewport/sceneIndex/fvpPathInterface.h"
+
+//Hydra headers
+#include <pxr/base/tf/declarePtrs.h>
```

Comment:
Are we using this header?

Reply:
Yes in renderoverride.h we have a member :
Fvp::BlockPrimRemovalPropagationSceneIndexRefPtr  _blockPrimRemovalPropagationSceneIndex;

Reply:
Right, I removed it, thanks.